crates/esthri/src/rusoto.rs

```diff
@@ -98,6 +99,49 @@ where
     }
 }
 
+pub enum S3StorageClass {
+    Standard,
+    StandardIA,
+    IntelligentTiering,
+    OneZoneIA,
+    GlacialInstantRetrieval,
+    GlacialFlexibleRetrieval,
+    GlacialDeepArchive,
+    RRS,
+}
+
+impl S3StorageClass {
```

Comment:
There might be other ways to do it, but a slightly safer & more idiomatically rust way to do this would be to do:

```rust
impl From<S3StorageClass> for String {
    fn from(s: S3StorageClass) -> String {
        match s {
            Standard => "STANDARD",
            StandardIA => "STANDARD_IA",
            IntelligentTiering => "INTELLIGENT_TIERING",
            OneZoneIA => "ONEZONE_IA",
            GlacialInstantRetrieval => "GLACIAL_INSTANT_RETRIEVAL",
            GlacialFlexibleRetrieval => "GLACIAL_FLEXIBLE_RETRIEVAL",
            GlacialDeepArchive => "GLACIAL_DEEP_ARCHIVE",
            RRS => "REDUCED_REDUNDANCY",
        }.into()
    }
}

impl TryFrom<&str> for S3StorageClass {
    type Error = Error;

    fn try_from(s: &str) -> Result<Self> {
        match s {
            "STANDARD" => Ok(Standard),
            "STANDARD_IA" => Ok(StandardIA),
            "INTELLIGENT_TIERING" => Ok(IntelligentTiering),
            "ONEZONE_IA" => Ok(OneZoneIA),
            "GLACIAL_INSTANT_RETRIEVAL" => Ok(GlacialInstantRetrieval),
            "GLACIAL_FLEXIBLE_RETRIEVAL" => Ok(GlacialFlexibleRetrieval),
            "GLACIAL_DEEP_ARCHIVE" => Ok(GlacialDeepArchive),
            "REDUCED_REDUNDANCY" => Ok(RRS),
            _ =>  Err(Error::UnknownStorageClass(s.to_string())),
        }
    }
}
```